modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/index.js

```diff
@@ -31,20 +34,49 @@ const Pages = () => {
 		dispatch,
 	] = useOnboarding();
 
-	const invitesPageHandle = () => {
-		const hasSubscriptionsDXPCloud = !!subscriptionGroups?.length;
+	const subscriptionDXPCloud = subscriptionGroups?.find(
+		(subscriptionGroup) => subscriptionGroup.name === PRODUCT_TYPES.dxpCloud
+	);
+
+	const subscriptionAnalyticsCloud = subscriptionGroups?.find(
+		(subscriptionGroup) =>
+			subscriptionGroup.name === PRODUCT_TYPES.analyticsCloud
+	);
 
-		if (hasSubscriptionsDXPCloud && !dxpCloudActivationSubmittedStatus) {
-			dispatch({
+	const invitesPageHandle = () => {
+		if (subscriptionDXPCloud && !dxpCloudActivationSubmittedStatus) {
+			return dispatch({
 				payload: ONBOARDING_STEP_TYPES.dxpCloud,
 				type: actionTypes.CHANGE_STEP,
 			});
 		}
-		else {
-			window.location.href = PAGE_ROUTER_TYPES.project(
-				project.accountKey
-			);
+		if (
+			subscriptionAnalyticsCloud &&
+			!analyticsCloudActivationSubmittedStatus
+		) {
+			return dispatch({
+				payload: ONBOARDING_STEP_TYPES.analyticsCloud,
+				type: actionTypes.CHANGE_STEP,
+			});
 		}
+		window.location.href = PAGE_ROUTER_TYPES.project(project.accountKey);
+	};
```

Comment:
Should be good use React Router here, to centralize the logic to redirect pages :)

Reply:
Hey man, is the approach that we already use on the onboarding flow, I just replied for the others steps